e2e-testing/cypress/e2e/12-data-view/data-view-search.cy.js

```diff
@@ -0,0 +1,18 @@
+import DataViewFunctions from '../../functions/DataViewFunctions/DataViewFunctions';
+const dataViewFunctions = new DataViewFunctions();
+
+beforeEach(() => {
```

Comment:
Could you put the before each in the describe block

Reply:
Could you also add the login to the before each block

Reply:
For the login, I thought we wouldn't need it since a user doesn't have to be logged in to search posts....

Reply:
@felabel actually that makes sense